lib/composites/menu/events/main.js

```diff
@@ -19,7 +19,8 @@ module.exports = (elements) => {
 
   document.addEventListener('click', (e) => {
     const target = e.target;
-    const isWithinDropdown = closest(target, '.dqpl-top-bar li[data-dropdown-expanded]');
+    const isWithinDropdown = closest(target, '.dqpl-top-bar [data-dropdown-expanded]');
```

Comment:
what is the reasoning here for removing li from the selector?

Reply:
No reason, forgot to add it back after removing it to test something. Good catch.